kolibri/plugins/coach/assets/src/views/coach-exercise-render-page/question-attempt.vue

```diff
@@ -36,19 +53,54 @@
     },
     data: () => ({
       selectedIndex: 0,
+      currPage: 1,
     }),
+    computed: {
+      numAttemptsShown() {
+        return Math.floor(this.elSize.width / 70) - 2;
+      },
+      currAttemptsShown() {
+        if (this.interaction_history.length <= ((this.currPage - 1) * this.numAttemptsShown)) {
+          // when expand, guarantee that no empty page shown.
+          this.currPage -= 1;
+        }
+        return this.interaction_history.slice((this.currPage - 1) * this.numAttemptsShown,
+          this.currPage * this.numAttemptsShown);
+      },
+    },
     methods: {
       attemptsText(number) {
         return this.$tr('attempts', { number });
       },
+      _ordinal_suffix_of(i) {
```

Comment:
We should use [selectordinal-format](https://formatjs.io/guides/message-syntax/#selectordinal-format) cc @indirectlylit